app/handlers/sessions_create.rb

```diff
@@ -71,6 +71,10 @@ def handle_during_login
       return :mismatched_authentication
     end
 
+    if signup_state.present?
+      run(TransferSignupState, signup_state: signup_state, user: authentication_user)
+    end
```

Comment:
@nathanstitt your earlier idea that we needed to add code to set a user's UUID during login was correct. (the stuff you put in  `SignupState.on_authentication...`).  In that PR you saw that the UUIDs were getting added and took that code out.  They were getting added because of this bug where users were going down the signup flow during login.  Because they are now correctly going through this `handle_during_login` method, we need to transfer the UUIDs once again.

Reply:
What a great description of a trail of errors that somehow was still working 😭